docs/en_us/platform_api/Makefile

```diff
@@ -1,164 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
@andy-armstrong Are you considering your questions about the future of API docs for edx-platform a blocker for this PR, or are you just hoping we will make progress on that front?